tests/integration_tests/functional/test_mmds.py

```diff
@@ -116,6 +118,31 @@ def test_custom_ipv4(test_microvm_with_ssh, network_config):
     _, stdout, _ = ssh_connection.execute_command(cmd)
     assert json.load(stdout) == data_store['latest']['meta-data']
 
+@pytest.mark.timeout(1200)
+def test_big_buffer(test_microvm_with_ssh, network_config):
+    """
+    Test send large buffer.
+
+    @type: functional
+    """
+    test_microvm = test_microvm_with_ssh
+    test_microvm.spawn()
+
+    response = test_microvm.mmds.get()
+    assert test_microvm.api_session.is_status_ok(response.status_code)
+    assert response.json() == {}
+
+    data_store = {
+        'latest': {
+            'meta-data': {
+            }
+        }
+    }
+
+    chars = ''.join(['b' for i in range(190*1024*1024)])
+    data_store["latest"]["meta-data"]["ami-id"] = chars
+    response = test_microvm.mmds.put(json=data_store)
+    assert test_microvm.api_session.is_status_no_content(response.status_code)
```

Comment:
this doesn't test that the data is actually available from the guest. I guess this is fine is we update the description to explicitly say that we are checking for seccomp errors as a regression test

Reply:
This test should eventually be removed/replaced when we actually fix the non-seccomp side of the issue and we have functional tests for MMDS, so I don't think it's a problem in this PR.

Reply:
hmm but could this be exercised for any API endpoint? Maybe this would be done for all huge JSON requests (even invalid?) I assume the mmap comes from a `malloc` or something, after the `recvfrom()`, but not sure

Reply:
it is indeed correct the issue can be triggered for any api endpoint. Maybe I should move it in test_api.py? wdyt @alindima  @georgepisaltu

Reply:
@AlexandruCihodaru I think that's the right thing to do. Please also update the name and description.